### PR TITLE
feature/AKD-33: Added 'VPN Tunnel state' plugin and test cases

### PR DESCRIPTION
**Pull request by AkhtarAmir**

### CODE REVIEW

File: plugins/aws/ec2/vpnTunnelState.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'VPN Tunnel State',
+    category: 'EC2',
+    description: 'Ensures that each AWS Virtual Private Network (VPN) connection has all tunnels up.',
+    more_info: 'AWS Virtual Private Network (VPN) should have tunnels up to ensure network traffic flow over Virtual Private Network.',
+    link: 'https://docs.aws.amazon.com/vpn/latest/s2svpn/VPNTunnels.html',
+    recommended_action: 'Establish a successfull VPN connection using IKE or IPsec configuration',
```
Comment (giorod3) [inline plugins/aws/ec2/vpnTunnelState.js:10]:
```suggestion
    recommended_action: 'Establish a successful VPN connection using IKE or IPsec configuration',
```

**Review by giorod3 — CHANGES_REQUESTED** [review]
this should be an opt-in type of plugin for customers using Site-to-Site VPN connections.

**Comment by AkhtarAmir** [comment]
I've added the setting to make this opt-in

**Review by giorod3 — APPROVED** [review]

File: helpers/aws/regions_china.js [hunk 2]
```diff
@@ -40,6 +40,7 @@ module.exports = {
     es: regions,
     kinesis: [],
     firehose: regions,
+    glue, regions,
```
Comment (giorod3) [inline helpers/aws/regions_china.js:43]:
```suggestion
    glue: regions,
```

**Pull request merged**